testing/kokoro/configure_cloudsql.bash

```diff
@@ -28,6 +28,6 @@ mv cloud_sql_proxy.linux.amd64 /cloud_sql_proxy
 chmod +x /cloud_sql_proxy
 mkdir /cloudsql && chmod 0777 /cloudsql
 
-/cloud_sql_proxy -instances="${MYSQL_INSTANCE}"=tcp:3306,${MYSQL_INSTANCE} -dir /cloudsql
-/cloud_sql_proxy -instances="${POSTGRES_INSTANCE}"=tcp:5432,${POSTGRES_INSTANCE} -dir /cloudsql
-/cloud_sql_proxy -instances="${SQLSERVER_INSTANCE}"=tcp:1433
+/cloud_sql_proxy -instances="${MYSQL_INSTANCE}"=tcp:3306,${MYSQL_INSTANCE} -dir /cloudsql &
+/cloud_sql_proxy -instances="${POSTGRES_INSTANCE}"=tcp:5432,${POSTGRES_INSTANCE} -dir /cloudsql &
+/cloud_sql_proxy -instances="${SQLSERVER_INSTANCE}"=tcp:1433 &
```

Comment:
This will return _immediately_ and the proxy might not be ready yet. You probably want to sleep for a few seconds and retry your connection for up to 30s.

Reply:
As per our offline chat, went with a 5s sleep and a comment for posterity, should this race condition ever wind up not in our favor